test/e2e/storage/testsuites/provisioning.go

```diff
@@ -175,6 +175,7 @@ func (p *provisioningTestSuite) DefineTests(driver storageframework.TestDriver,
 			ExpectedSize: claimSize,
 			VolumeMode:   pattern.VolMode,
 		}
+		l.testCase.NodeSelection = l.config.ClientNodeSelection
```

Comment:
nit: this could be merged in the previous struct definition

Reply:
Done.